src/main/java/org/opentripplanner/updater/vehicle_sharing/parking_zones/ParkingZonesUpdater.java

```diff
@@ -20,14 +15,13 @@
 import java.util.Objects;
 
 import static java.util.function.Function.identity;
-import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toMap;
 
 public class ParkingZonesUpdater extends PollingGraphUpdater {
 
     private static final Logger LOG = LoggerFactory.getLogger(ParkingZonesUpdater.class);
 
-    private final ParkingZonesGetter parkingZonesGetter = new ParkingZonesGetter();
+    protected ParkingZonesGetter parkingZonesGetter = new ParkingZonesGetter();
```

Comment:
Co się stało z final?

Reply:
Nie mamy dependency injection (a przynajmniej o tym nie wiem xD), a ja musiałem jakoś w testach wstawić zmockowany `ParkingZonesGetter`. Paskudne rozwiązanie, wiem - jeśli ktoś zna jakieś lepsze, to chętnie poznam 👍